components/__tests__/Currency.test.js

```diff
@@ -10,8 +10,8 @@ describe('Currency', () => {
     snapshot(<Currency value={1900000} currency="USD" />, { IntlProvider: { locale: 'fr' } });
   });
 
-  it('abbreviated version', () => {
-    snapshot(<Currency abbreviate value={4200} currency="USD" />);
-    snapshot(<Currency abbreviate value={1900000} currency="USD" />, { IntlProvider: { locale: 'fr' } });
+  it('currency format with separators', () => {
+    snapshot(<Currency formatWithSeparators value={4200} currency="USD" />);
+    snapshot(<Currency formatWithSeparators value={1900000} currency="USD" />, { IntlProvider: { locale: 'fr' } });
```

Comment:
Would be great to add some test cases to show the variations across different locales

```suggestion
    snapshot(<Currency formatWithSeparators value={1900000} currency="USD" />, { IntlProvider: { locale: 'fr' } });
    snapshot(<Currency formatWithSeparators value={1900000} currency="USD" />, { IntlProvider: { locale: 'en-US' } });
```

Reply:
Thanks. Done 👍🏽